src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2837,4 +2837,13 @@ We use App Center to track which settings are being used, find bugs, and fix cra
   <data name="RestoreSelection" xml:space="preserve">
     <value>Restore</value>
   </data>
+  <data name="ShortcutDialog.Title" xml:space="preserve">
+    <value>Shortcut cannot be opened</value>
+  </data>
+  <data name="ShortcutDialog.Delete" xml:space="preserve">
+    <value>Do you want to delete this shortcut?</value>
+  </data>
+  <data name="ShortcutDialog.Text" xml:space="preserve">
+    <value>The target destination cannot be found {0}.</value>
```

Comment:
Can this be merged with the description string?

Reply:
Perfect! Maybe the primary button should say delete, what do you think?

Reply:
I was thinking that but I'm not sure. Do we have other primary buttons that describe the action?

Reply:
The confirm delete modal has a "delete" button.

Reply:
Alright I’ll keep it consistent and change the primary button to delete.